x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('when on the managing page', () => {
           <I18nProvider>
             <Router history={history}>
               <RouteCapture>
-                <ManagementList />
+                <HostList />
```

Comment:
My guess is that the jest tests are failing right now because we need to wrap this in ` <EuiThemeProvider />` like we do here https://github.com/parkiino/kibana/blob/task/host-enhancements/x-pack/plugins/endpoint/public/applications/endpoint/index.tsx#L60